mapper/mapper.go

```diff
@@ -327,6 +327,13 @@ func CreateVersionsList(ctx context.Context, d dataset.Model, edition dataset.Ed
 
 		p.Data.Versions = append(p.Data.Versions, version)
 	}
+
+	for i, ver := range p.Data.Versions{
+		if ver.VersionNumber == latestVersionNumber {
+			p.Data.Versions[i].IsLatest = true
+		}
+	}
+
```

Comment:
👇 I know its not in your code but WTF is that?! We should be using a proper sort function here.

Reply:
This was taken from the official GoLang wiki https://github.com/golang/go/wiki/SliceTricks reverse splice. 

Have replaced it with a sort function instead, worth noting I believe that it will perform a little slower on dataset pages that have hundreds or thousands of versions in comparison to the reverse.